lightly/api/api_workflow_tags.py

```diff
@@ -156,12 +161,13 @@ def create_tag_from_filenames(
             'name': new_tag_name, 
             'prevTagId': parent_tag_id, 
             'bitMaskData': bitmask.to_hex(), 
-            'totSize': tot_size
+            'totSize': tot_size,
+            'creator': TagCreator.USER_PIP,
         }
```

Comment:
Why don't we create an instance of `TagCreateRequest`?

Reply:
I guess it is because of some api generator limitation.